limitador/src/lib.rs

```diff
@@ -306,9 +309,9 @@ impl AsyncRateLimiterBuilder {
 }
 
 impl RateLimiter {
-    pub fn new() -> Self {
+    pub fn new(cache_size: u64) -> Self {
```

Comment:
Ok, I kind of understand now why we need the cache size and can't use the default I asked before..

Reply:
Yes... there needs to be a value provided (and see the other comment on why I did it this way)